astro/src/content/docs/release-notes/index.mdx

```diff
@@ -47,6 +47,19 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
 
 Looking for release notes older than 1.44.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.60.2' releaseDate='September 26, 2025' />
+
+### Known Issues
+
+* <KnownIssue issue="3186">Performing a user reindex occasionally results in an email, username, or phoneNumber, for a small fraction of users, to not be searchable via Elasticsearch/OpenSearch.</KnownIssue>
```

Comment:
We carried this through on `1.60.0` and `1.60.1`. Do I need to add the resolved SAML Issue to `1.60.1` in addition to `1.60.0`?

Hopefully it's rare that known issues are listed in more than one version. This SAML issue wasn't identified until `1.60.1` was already released, and this reindex issue is targeted for `1.61.0`.